src/test/scala/fastaHeaders.scala

```diff
@@ -0,0 +1,18 @@
+package ohnosequences.db.rna16s.test
+
+import ohnosequences.api.rnacentral.Entry
+import ohnosequences.fastarious._, fasta._
+
+case object fastaFormat {
+
+  val joinDescriptions: Entry => String =
+    _.sequenceAnnotations.map(_.description).mkString(" | ")
+
+  // TODO add DB ID and version
```

Comment:
Why do you want DB ID and version here?

Reply:
I no longer remember :) Let's remove it.